<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>174</width>
     <height>305</height>
    </rect>
   </property>
   <property name="title">
    <string>Personality</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="NameR">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NameRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="SurnameR">
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SurnameRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="TelR">
        <property name="text">
         <string>Telephone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TelRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="MailR">
        <property name="text">
         <string>Mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MailRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="CityR">
        <property name="text">
         <string>City</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CityRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="StreetR">
        <property name="text">
         <string>Street</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StreetRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="HouseR">
        <property name="text">
         <string>House</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="HouseRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="FlatR">
        <property name="text">
         <string>Flat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FlatRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="IndexR">
        <property name="text">
         <string>Index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IndexRE"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>162</width>
     <height>155</height>
    </rect>
   </property>
   <property name="title">
    <string>Card</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NamecardRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="NumberR">
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NumberRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="DateR">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DateRE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="CVCR">
        <property name="text">
         <string>CVC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CVCRE"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
